[2024-03-15T00:37:00.519-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: PII_Data_Detection.missing_values_removal scheduled__2024-03-15T04:30:00+00:00 [queued]>
[2024-03-15T00:37:00.526-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: PII_Data_Detection.missing_values_removal scheduled__2024-03-15T04:30:00+00:00 [queued]>
[2024-03-15T00:37:00.526-0400] {taskinstance.py:2193} INFO - Starting attempt 3 of 3
[2024-03-15T00:37:00.551-0400] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): missing_values_removal> on 2024-03-15 04:30:00+00:00
[2024-03-15T00:37:00.558-0400] {standard_task_runner.py:60} INFO - Started process 137819 to run task
[2024-03-15T00:37:00.562-0400] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'PII_Data_Detection', 'missing_values_removal', 'scheduled__2024-03-15T04:30:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/src/airflow.py', '--cfg-path', '/tmp/tmppyum1ulc']
[2024-03-15T00:37:00.564-0400] {standard_task_runner.py:88} INFO - Job 62: Subtask missing_values_removal
[2024-03-15T00:37:00.726-0400] {task_command.py:423} INFO - Running <TaskInstance: PII_Data_Detection.missing_values_removal scheduled__2024-03-15T04:30:00+00:00 [running]> on host MSI.
[2024-03-15T00:37:00.991-0400] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='PII_Data_Detection' AIRFLOW_CTX_TASK_ID='missing_values_removal' AIRFLOW_CTX_EXECUTION_DATE='2024-03-15T04:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-15T04:30:00+00:00'
[2024-03-15T00:37:00.994-0400] {logging_mixin.py:188} INFO - fetched project directory successfully /home/vineshgvk/PII-Data
[2024-03-15T00:37:01.020-0400] {logging_mixin.py:188} INFO - fetched path from load_gcp_data task /home/vineshgvk/PII-Data/dags/processed/train_com.json
[2024-03-15T00:37:01.022-0400] {logging_mixin.py:188} INFO - Loading data from: /home/vineshgvk/PII-Data/dags/processed/train_com.json
[2024-03-15T00:37:01.069-0400] {logging_mixin.py:188} INFO - created outputPath /home/vineshgvk/PII-Data/data/processed/missing_values.pkl
[2024-03-15T00:37:01.071-0400] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/vineshgvk/PII-Data/dags/src/missing_values.py", line 50, in naHandler
    with open(outputPath, "wb") as file:
FileNotFoundError: [Errno 2] No such file or directory: '/home/vineshgvk/PII-Data/data/processed/missing_values.pkl'
[2024-03-15T00:37:01.090-0400] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=PII_Data_Detection, task_id=missing_values_removal, execution_date=20240315T043000, start_date=20240315T043700, end_date=20240315T043701
[2024-03-15T00:37:01.154-0400] {standard_task_runner.py:107} ERROR - Failed to execute job 62 for task missing_values_removal ([Errno 2] No such file or directory: '/home/vineshgvk/PII-Data/data/processed/missing_values.pkl'; 137819)
[2024-03-15T00:37:01.184-0400] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-15T00:37:01.231-0400] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
